### PR TITLE
Add French translation: french README.md

### PR DESCRIPTION
**Pull request by ThibaultLengagne**
# What does this PR do?

Add the French version of README.md

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@stevhliu and @MKhalusova

### CODE REVIEW

File: README_fr.md [hunk 1]
```diff
@@ -0,0 +1,573 @@
...
+- [Traduction avec T5](https://huggingface.co/t5-base?text=My+name+is+Wolfgang+and+I+live+in+Berlin)
+
+En vision par ordinateur :
+- [Classification d'images avec ViT](https://huggingface.co/google/vit-base-patch16-224)
+- [Détection d'objets avec DETR](https://huggingface.co/facebook/detr-resnet-50)
+
```
Comment (stevhliu) [inline README_fr.md:102]:
```suggestion
```

File: README_fr.md [hunk 2]
```diff
@@ -0,0 +1,573 @@
...
+```
+
+Ici, nous obtenons une liste d'objets détectés dans l'image, avec une boîte entourant l'objet et un score de confiance. Voici l'image originale à gauche, avec les prédictions affichées à droite :
+
+<h3 align="center">
+    <a><img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/coco_sample.png" width="400"></a>
+    <a><img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/coco_sample_post_processed.png" width="400"></a>
+</h3>
+
+Vous pouvez en savoir plus sur les tâches supportées par l'API pipeline dans ce tutoriel.
```
Comment (stevhliu) [inline README_fr.md:190]: Missing link to the [task summary](https://huggingface.co/docs/transformers/task_summary)

File: README_fr.md [hunk 3]
```diff
@@ -0,0 +1,573 @@
...
+tokenizer = AutoTokenizer.from_pretrained("bert-base-uncased")
+model = TFAutoModel.from_pretrained("bert-base-uncased")
+
+inputs = tokenizer("Bonjour le monde !", return_tensors="tf")
+outputs = model(**inputs)
+```
+
+Le tokenizer est responsable de toutes les étapes de prétraitement que le modèle préentraîné attend et peut être appelé directement sur une seule chaîne de caractères (comme dans les exemples ci-dessus) ou sur une liste. Il produira un dictionnaire que vous pouvez utiliser dans votre code ou simplement passer directement à votre modèle en utilisant l'opérateur de déballage **.
+
+Le modèle lui-même est un module nn.Module PyTorch ou un modèle tf.keras.Model TensorFlow (selon votre backend) que vous pouvez utiliser comme d'habitude. Ce tutoriel explique comment intégrer un tel modèle dans une boucle d'entraînement classique PyTorch ou TensorFlow, ou comment utiliser notre API Trainer pour affiner rapidement sur un nouvel ensemble de données.
```
Comment (stevhliu) [inline README_fr.md:218]: Missing links to [`nn.Module`](https://pytorch.org/docs/stable/nn.html#torch.nn.Module), [`tf.keras.Model`](https://www.tensorflow.org/api_docs/python/tf/keras/Model), and the [training tutorial](https://huggingface.co/docs/transformers/training).

File: README_fr.md [hunk 4]
```diff
@@ -0,0 +1,573 @@
...
+### Avec pip
+
+Ce référentiel est testé sur Python 3.8+, Flax 0.4.1+, PyTorch 1.11+ et TensorFlow 2.6+.
+
+Vous devriez installer 🤗 Transformers dans un [environnement virtuel](https://docs.python.org/3/library/venv.html). Si vous n'êtes pas familier avec les environnements virtuels Python, consultez le [guide utilisateur](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/).
+
+D'abord, créez un environnement virtuel avec la version de Python que vous allez utiliser et activez-le.
+
+Ensuite, vous devrez installer au moins l'un de Flax, PyTorch ou TensorFlow.
+Veuillez vous référer à la page d'installation de [TensorFlow](https://www.tensorflow.org/install/), de [PyTorch](https://pytorch.org/get-started/locally/#start-locally) et/ou de [Flax]](https://github.com/google/flax#quick-install) et [Jax](https://github.com/google/jax#installation) pour connaître la commande d'installation spécifique à votre plateforme.
```
Comment (stevhliu) [inline README_fr.md:260]:
```suggestion
Veuillez vous référer à la page d'installation de [TensorFlow](https://www.tensorflow.org/install/), de [PyTorch](https://pytorch.org/get-started/locally/#start-locally) et/ou de [Flax](https://github.com/google/flax#quick-install) et [Jax](https://github.com/google/jax#installation) pour connaître la commande d'installation spécifique à votre plateforme.
```

File: README_fr.md [hunk 5]
```diff
@@ -0,0 +1,573 @@
...
+
+Lorsqu'un de ces backends est installé, 🤗 Transformers peut être installé avec pip comme suit :
+
+```bash
+pip install transformers
+```
+
+Si vous souhaitez jouer avec les exemples ou avez besoin de la dernière version du code et ne pouvez pas attendre une nouvelle version, vous devez installer la bibliothèque à partir de la source.
```
Comment (stevhliu) [inline README_fr.md:268]: Missing link to installation [from source](https://huggingface.co/docs/transformers/installation#installing-from-source).

**Review by stevhliu — COMMENTED** [review]
LGTM aside from some missing links! I don't speak any French so I can't review the content. Do you know any other French speakers who would be interested in contributing and reviewing your work?

File: README_fr.md [hunk 6]
```diff
@@ -0,0 +1,573 @@
...
+
+<h3 align="center">
+    <a href="https://hf.co/course"><img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/course_banner.png"></a>
+</h3>
+
+🤗 Transformers fournit des milliers de modèles pré-entraînés pour effectuer des tâches sur différentes modalités telles que le texte, la vision et l'audio.
+
+Ces modèles peuvent être appliqués à :
+
+* 📝 Texte, pour des tâches telles que la classification de texte, l'extraction d'informations, la réponse aux questions, la résumé, la traduction et la génération de texte, dans plus de 100 langues.
```
Comment (Sarapuce) [inline README_fr.md:74]:
```suggestion
* 📝 Texte, pour des tâches telles que la classification de texte, l'extraction d'informations, la réponse aux questions, le résumé, la traduction et la génération de texte, dans plus de 100 langues.
```

File: README_fr.md [hunk 7]
```diff
@@ -0,0 +1,573 @@
...
+* 🖼️ Images, pour des tâches telles que la classification d'images, la détection d'objets et la segmentation.
+* 🗣️ Audio, pour des tâches telles que la reconnaissance vocale et la classification audio.
+
+Les modèles de transformer peuvent également effectuer des tâches sur **plusieurs modalités combinées**, telles que la réponse aux questions sur table, la reconnaissance optique de caractères, l'extraction d'informations à partir de documents numérisés, la classification vidéo et la réponse aux questions visuelles.
```
Comment (Sarapuce) [inline README_fr.md:78]:
```suggestion
Les modèles de transformer peuvent également effectuer des tâches sur **plusieurs modalités combinées**, telles que la réponse aux questions sur des tableaux, la reconnaissance optique de caractères, l'extraction d'informations à partir de documents numérisés, la classification vidéo et la réponse aux questions visuelles.
```

File: README_fr.md [hunk 8]
```diff
@@ -0,0 +1,573 @@
...
+
+🤗 Transformers fournit des API pour télécharger et utiliser rapidement ces modèles pré-entraînés sur un texte donné, les affiner sur vos propres ensembles de données, puis les partager avec la communauté sur notre [hub de modèles](https://huggingface.co/models). En même temps, chaque module Python définissant une architecture est complètement indépendant et peut être modifié pour permettre des expériences de recherche rapides.
+
+🤗 Transformers est soutenu par les trois bibliothèques d'apprentissage profond les plus populaires — [Jax](https://jax.readthedocs.io/en/latest/), [PyTorch](https://pytorch.org/) et [TensorFlow](https://www.tensorflow.org/) — avec une intégration transparente entre eux. Il est facile de former vos modèles avec l'un avant de les charger pour l'inférence avec l'autre.
+
+## Démos en ligne
+
+Vous pouvez tester la plupart de nos modèles directement sur leurs pages du [hub de modèles](https://huggingface.co/models). Nous proposons également [l'hébergement privé de modèles, la version et une API d'inférence](https://huggingface.co/pricing) pour des modèles publics et privés.
```
Comment (Sarapuce) [inline README_fr.md:86]:
```suggestion
Vous pouvez tester la plupart de nos modèles directement sur leurs pages du [hub de modèles](https://huggingface.co/models). Nous proposons également [l'hébergement privé de modèles, le versionning et une API d'inférence](https://huggingface.co/pricing) pour des modèles publics et privés.
```

File: README_fr.md [hunk 9]
```diff
@@ -0,0 +1,573 @@
...
+- [Réponses aux questions sur table avec TAPAS](https://huggingface.co/google/tapas-base-finetuned-wtq)
+- [Réponses aux questions visuelles avec ViLT](https://huggingface.co/dandelin/vilt-b32-finetuned-vqa)
+- [Classification d'images sans étiquette avec CLIP](https://huggingface.co/openai/clip-vit-large-patch14)
+- [Réponses aux questions sur les documents avec LayoutLM](https://huggingface.co/impira/layoutlm-document-qa)
+- [Classification vidéo sans étiquette avec X-CLIP](https://huggingface.co/docs/transformers/model_doc/xclip)
+
+
+## 100 projets utilisant Transformers
+
+Transformers sont plus qu'une boîte à outils pour utiliser des modèles pré-entraînés : c'est une communauté de projets construits autour d'elle et du Hub Hugging Face. Nous voulons que Transformers permette aux développeurs, chercheurs, étudiants, professeurs, ingénieurs et à quiconque d'imaginer et de réaliser leurs projets de rêve.
```
Comment (Sarapuce) [inline README_fr.md:124]:
```suggestion
Transformers est plus qu'une boîte à outils pour utiliser des modèles pré-entraînés : c'est une communauté de projets construits autour de lui et du Hub Hugging Face. Nous voulons que Transformers permette aux développeurs, chercheurs, étudiants, professeurs, ingénieurs et à quiconque d'imaginer et de réaliser leurs projets de rêve.
```

File: README_fr.md [hunk 10]
```diff
@@ -0,0 +1,573 @@
...
+
+# Allouer un pipeline pour l'analyse de sentiment
+>>> classifieur = pipeline('sentiment-analysis')
+>>> classifieur('Nous sommes très heureux d'introduire le pipeline dans le référentiel transformers.')
+[{'label': 'POSITIF', 'score': 0.9996980428695679}]
+```
+
+La deuxième ligne de code télécharge et met en cache le modèle pré-entraîné utilisé par le pipeline, tandis que la troisième l'évalue sur le texte donné. Ici, la réponse est "positive" avec une confiance de 99,97%.
+
+De nombreuses tâches ont un pipeline pré-entraîné prêt à l'emploi, en NLP, mais aussi en vision par ordinateur et en parole. Par exemple, nous pouvons facilement extraire les objets détectés dans une image :
```
Comment (Sarapuce) [inline README_fr.md:151]:
```suggestion
De nombreuses tâches ont une pipeline pré-entraîné prêt à l'emploi, en NLP, mais aussi en vision par ordinateur et en parole. Par exemple, nous pouvons facilement extraire les objets détectés dans une image :
```

File: README_fr.md [hunk 11]
```diff
@@ -0,0 +1,573 @@
...
+    - Une API unifiée pour utiliser tous nos modèles préentraînés.
+
+1. Coûts informatiques réduits, empreinte carbone plus petite :
+    - Les chercheurs peuvent partager des modèles entraînés au lieu de toujours les réentraîner.
+    - Les praticiens peuvent réduire le temps de calcul et les coûts de production.
+    - Des dizaines d'architectures avec plus de 400 000 modèles préentraînés dans toutes les modalités.
+
+1. Choisissez le bon framework pour chaque partie de la vie d'un modèle :
+    - Entraînez des modèles de pointe en 3 lignes de code.
+    - Déplacez un seul modèle entre les frameworks TF2.0/PyTorch/JAX à volonté.
```
Comment (Sarapuce) [inline README_fr.md:235]:
```suggestion
    - Transférer un seul modèle entre les frameworks TF2.0/PyTorch/JAX à volonté.
```

"Déplacer" doesn't really have the original meaning of the sentence in French

File: README_fr.md [hunk 12]
```diff
@@ -0,0 +1,573 @@
...
+
+## En savoir plus
+
+| Section | Description |
+|-|-|
+| [Documentation](https://huggingface.co/docs/transformers/) | Documentation complète de l'API et tutoriels |
+| [Résumé des tâches](https://huggingface.co/docs/transformers/task_summary) | Tâches prises en charge par les 🤗 Transformers |
+| [Tutoriel de prétraitement](https://huggingface.co/docs/transformers/preprocessing) | Utilisation de la classe `Tokenizer` pour préparer les données pour les modèles |
+| [Entraînement et ajustement fin](https://huggingface.co/docs/transformers/training) | Utilisation des modèles fournis par les 🤗 Transformers dans une boucle d'entraînement PyTorch/TensorFlow et de l'API `Trainer` |
+| [Tour rapide : Scripts d'ajustement fin/utilisation](https://github.com/huggingface/transformers/tree/main/examples) | Scripts d'exemple pour ajuster finement les modèles sur une large gamme de tâches |
```
Comment (Sarapuce) [inline README_fr.md:555]:
```suggestion
| [Tour rapide : Scripts d'ajustement fin/utilisation](https://github.com/huggingface/transformers/tree/main/examples) | Scripts d'exemple pour ajuster finement les modèles sur une large gamme de tâches |
```
```suggestion
| [Tour rapide : Scripts d'ajustement fin/d'utilisation](https://github.com/huggingface/transformers/tree/main/examples) | Scripts d'exemple pour ajuster finement les modèles sur une large gamme de tâches |
```

More comprehensible like this

**Review by Sarapuce — COMMENTED** [review]
Hello, I can help with the French translation. Here are some modifications that can be made to improve the comprehension of French speakers.

**Comment by ThibaultLengagne** [comment]
Hello,

Thanks @stevhliu and @Sarapuce for the review. I have fixed every remark. 

Seems good to go :+1:

**Review by stevhliu — APPROVED** [review]
Great work @ThibaultLengagne and thanks for reviewing @Sarapuce! 🎉

**Pull request merged**